src/CampaignKit.WorldMap/Entities/Tile.cs

```diff
@@ -11,84 +12,75 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-using System;
-using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
+// </copyright>
 
 namespace CampaignKit.WorldMap.Entities
 {
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    using Microsoft.Azure.Cosmos.Table;
```

Comment:
Is Cosmos.Table the same as Storage.Tables?

Reply:
Yes.  They've consolidated the Azure Table and Cosmos DB client libraries:

https://devblogs.microsoft.com/azure-sdk/azure-tables-client-libraries/

![image](https://user-images.githubusercontent.com/17418008/110402081-bc4b8a80-8048-11eb-8a43-608fe48a30cf.png)

You can choose which service you want to use at deploy time.  Azure Tables for lightweight, Cosmos DB for heavyweight.  Microsoft wants to make it as easy as possible to charge you more $$$ if you need more performance.

Reply:
Thanks for that update!